targets/photos/web/public/index.styl

```diff
@@ -21,7 +21,7 @@
 
 .pho-content-header.--no-icon
     left 0
-    left-padded(15px)
+    padding-left em(15px)
```

Comment:
might as well make it `1rem` no ?

Reply:
Well, no. 16px is 1rem, but I could round it up.

Reply:
Yeah that's what I meant 😛 . I just think we should avoid using that `em` function.

Reply:
I know, I change those when I have the opportunity but I was more focused on repairing my mess than cleaning things up :D